e2etest/e2etest_suite_test.go

```diff
@@ -49,6 +49,7 @@ var (
 	ipv6ForContainers string
 	useOperator       bool
 	reportPath        string
+	Updater           testsconfig.Updater
```

Comment:
maybe i missing something here, but why it has to start with an uppercase letter if we only use it in this file?

Reply:
That's correct, it can be private

Reply:
fixed